client/editSchedule.js

```diff
@@ -110,10 +129,10 @@ setupCompetitionCalendar = function(template, $calendarDiv, $editModal) {
         $editModal.modal('show');
       },
       eventDrop: function(calEvent, delta, revertFunc, jsEvent, ui, view) { // Existing entry dragged
-        eventChanged(calEvent);
+        eventChanged(calEvent, revertFunc);
```

Comment:
Nevermind, I just saw `tests/mocha/server/CompetitionTest.js`. Looks like these cases *do* cause competition validation to fail. I'll need to see how these exceptions actually present themselves to the user.

I think there's still some part of me that's afraid we'll somehow end up with an orphaned event. The nice thing about the old behavior was that we allowed the user to do anything, and if they caused orphaned events, they were just considered unscheduled and would show up on the list of available events to schedule. I'm torn between wanting to add that back, and the fact that this would be code that (ideally) would never be exercised. Either way, this shouldn't block merging your code up.